setup.cfg

```diff
@@ -21,10 +21,12 @@ classifiers =
     Programming Language :: Python :: 3.6
```

Comment:
Python 3.6 is already EOL. See: https://endoflife.date/python
Should we remain this support?